internal/profiles/main_viper_test.go

```diff
@@ -0,0 +1,242 @@
+package profiles_test
+
+import (
+	"testing"
+
+	"github.com/pingidentity/pingctl/internal/profiles"
+	"github.com/pingidentity/pingctl/internal/testing/testutils_viper"
+)
+
+// Test GetMainViper function
+func TestGetMainViper(t *testing.T) {
+	testutils_viper.InitVipers(t)
+
+	v := profiles.GetMainViper()
+	if v == nil {
+		t.Errorf("GetMainViper returned nil")
+	}
+}
+
+// Test GetConfigActiveProfile function
+func TestGetConfigActiveProfile(t *testing.T) {
+	testutils_viper.InitVipers(t)
+
+	profile := profiles.GetConfigActiveProfile()
+	if profile == "" {
+		t.Errorf("GetConfigActiveProfile returned empty string")
+	}
+
+	if profile != "default" {
+		t.Errorf("GetConfigActiveProfile returned %s, expected 'default'", profile)
+	}
+}
+
+// Test SetConfigActiveProfile function
+func TestSetConfigActiveProfile(t *testing.T) {
+	testutils_viper.InitVipers(t)
+
+	err := profiles.SetConfigActiveProfile("production")
+	if err != nil {
+		t.Errorf("SetConfigActiveProfile returned error: %v", err)
+	}
+
+	profile := profiles.GetConfigActiveProfile()
+	if profile != "production" {
+		t.Errorf("GetConfigActiveProfile returned %s, expected 'test'", profile)
```

Comment:
Looks like this should say expected 'production'

Reply:
nice find! fixed